src/main/java/net/atos/zac/signalering/model/SignaleringType.java

```diff
@@ -49,10 +50,18 @@ public Type getType() {
         return type;
     }
 
+    public void setType(Type type) {
```

Comment:
Seemed like easiest way to allow fixture creation

Reply:
maybe make it protected, so that it's available for the tests, but not too broadly...

Reply:
Or would it be better through a constructor? Or builder?

Reply:
Changed this to package access.

Adding constructor or a builder is a good idea, but would need more changes where this class is used.